Added: alt attribute to few images (#19626)
mdn/content: files/en-us/learn/css/css_layout/practical_positioning_examples/index.md

In our second example, we will take our first example — our info-box — and add it into the context of a full web page. But not only that — we'll give it fixed position so that it stays in the same position in the browser window. When the main content scrolls, the info-box will stay in the same position on the screen. Our finished example will look like this:

> ![Info-box is a container with 3 tabs out of which the first tab is selected and only the contents of the first tab are displayed. It is given a fixed position. The info-box is jammed to the left-hand side top corner of the window. It's width is 452 pixels. A container with fake content for demonstration purposes occupies the right half of the window and has a height greater than the window height and is scrollable. When the page is scrolled, only the right-hand side container moves and the info-box will stay in the same position on the screen. ](fixed-info-box.png)

Reviewer: ```suggestion
![Info-box is a container with 3 tabs with the first tab selected and only the contents of the first tab are displayed. It is given a fixed position. The info-box is positioned at the top left corner of the window with a width of 452 pixels. A container of fake content occupies the rest right half of the window; the fake content container is taller than the window and is scrollable. When the page is scrolled, the right-hand side container moves while the info-box stays fixed in the same position on the screen. ](fixed-info-box.png)
```

Our finished example will look like this:

> ![The screen has a panel displaying some information on the right-hand side of it. A 'question mark' icon is jammed to the right-hand top corner of the window. The panel slides on and off the screen at the press of this 'question mark' icon.](hidden-sliding-panel.png)

Reviewer: ```suggestion
![A blank screen on the left 60% of the screen with a 40% width panel displaying information on the right. A 'question mark' icon is in the top-right corner. The panel slides on and off the screen at the press of this 'question mark' icon.](hidden-sliding-panel.png)
```
